### PR TITLE
Implement 'Your Blessing Skills are Disabled' for Essence Worm

### PR DESCRIPTION
**Pull request by Paliak**
Fixes https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/5112

### Description of the problem being solved:
Implements 'Your Blessing Skills are Disabled' for Essence Worm.

### CODE REVIEW

**Comment by QuickStick123** [comment]
I think there is only one variant of this that exists currently and that is the latest version typically we don't have unique versions in PoB that don't exist but we will support the legacy mods.

**Comment by Paliak** [comment]
Afaik the 'Your Blessing Skills are Disabled' mod was added in 3.17 so i thought i should add it as a variant https://www.poewiki.net/wiki/Essence_Worm

**Comment by QuickStick123** [comment]
This change effect exist items so we usually update all instances of the unique then

File: src/Data/Uniques/ring.lua [hunk 1]
```diff
@@ -449,14 +449,17 @@ Implicits: 1
...
 Variant: Current
 Requires Level 38
 Implicits: 1
 Has 1 Socket
 +2 to Level of Socketed Aura Gems
 Socketed Gems Have no Reservation
+{variant:3}Your Blessing Skills are Disabled
 {variant:1}80% reduced Mana Reservation Efficiency of Skills
 {variant:2}80% reduced Reservation Efficiency
+{variant:3}80% reduced Reservation Efficiency of Skills
```
Comment (QuickStick123) [inline src/Data/Uniques/ring.lua:462]:
```suggestion
Your Blessing Skills are Disabled
80% reduced Reservation Efficiency of Skills
```

File: src/Data/Uniques/ring.lua [hunk 2]
```diff
@@ -449,14 +449,17 @@ Implicits: 1
 Essence Worm
 Unset Ring
 Variant: Pre 3.16.0
+Variant: Pre 3.17.0
 Variant: Current
```
Comment (QuickStick123) [inline src/Data/Uniques/ring.lua:453]:
```suggestion
```

**Review by LocalIdentity — APPROVED** [review]

**Pull request merged**